modules/gapi/include/opencv2/gapi/garg.hpp

```diff
@@ -91,12 +90,11 @@ using GArgs = std::vector<GArg>;
 using GRunArg  = util::variant<
 #if !defined(GAPI_STANDALONE)
     cv::Mat,
-    cv::Scalar,
     cv::UMat,
```

Comment:
Why cv::Scalar was removed from here. We don't ever use Scalar from ocv anymore?

Reply:
If STANDALONE mode == OFF, then cv::Scalar is cv::Scalar (from OpenCV), else cv::Scalar is cv::gapi::own::Scalar. 
And yes, if STANDALONE mode == ON, then variant doesn't contain cv::Scalar(OpenCV).